x/liquidstaking/keeper/rebalancing.go

```diff
@@ -83,17 +83,21 @@ func (k Keeper) Rebalance(ctx sdk.Context, proxyAcc sdk.AccAddress, liquidVals t
 
 	failCount := 0
 	rebalancingThresholdAmt := rebalancingTrigger.Mul(totalLiquidTokens.ToDec()).TruncateInt()
-	for i := 0; i < liquidVals.Len(); i++ {
-		// sync totalLiquidTokens, liquidTokenMap applied rebalancing
-		var liquidTokenMap map[string]sdk.Int
-		totalLiquidTokens, liquidTokenMap = liquidVals.TotalLiquidTokens(ctx, k.stakingKeeper, false)
 
+	var liquidTokenMap map[string]sdk.Int
+	totalLiquidTokens, liquidTokenMap = liquidVals.TotalLiquidTokens(ctx, k.stakingKeeper, false)
```

Comment:
I see `liquidVals.TotalLiquidTokens(ctx, k.stakingKeeper, false)` is called at the start of the function. Can we use that function instead of calling again?

Reply:
That's possible. Thank you for finding out, let me fix